package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tforster/webproducer",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "description": "WebProducer, or simply WP, is a serverless application for publishing content to the web. It equally supports websites and web applications with a lean, streams based, architecture.",
   "author": "",
   "license": "ISC",
```

Comment:
Should this state the MIT license?

Reply:
Yes, can you add it as a suggested change?

Reply:
Nope, it won't let me. Perhaps because I completed my review. It only lets me edit the file directly and commit to the branch or create a new branch, no option to suggest edits.